### PR TITLE
Move the locked asset deposit value to the "Assets" page

### PR DESCRIPTION
**Pull request by neuodev**
The looked amount is now on the assets page. Will only be displayed for Cardano. 

Screen Shot 2021-09-28 at 1 28 24 PM
Screen Shot 2021-09-28 at 1 28 34 PM

### CODE REVIEW

File: packages/yoroi-extension/app/containers/wallet/WalletAssetsPage.js [hunk 1]
```diff
@@ -40,7 +42,19 @@ export default class WalletAssetsPage extends Component<InjectedOrGenerated<Gene
         }));
       })();
 
-    return <AssetsPage assetsList={assetsList} />
+      const txRequests = this.generated.stores.transactions.getTxRequests(publicDeriver);
+      const assetDeposit = txRequests.requests.getAssetDepositRequest.result || null;
+      const { stores } = this.generated;
+      const { profile } = stores;
+      const isNonZeroDeposit = !assetDeposit?.isEmpty();
+    return (
```
Comment (yushih) [inline packages/yoroi-extension/app/containers/wallet/WalletAssetsPage.js:50]: Please fix indentations.

File: packages/yoroi-extension/app/components/wallet/assets/AssetsList.js [hunk 2]
```diff
@@ -131,10 +142,50 @@ export default class AssetsList extends Component<Props, State> {
     return <ArrowsList />;
   }
 
+  renderAmountDisplay: {|
+    shouldHideBalance: boolean,
```
Comment (yushih) [inline packages/yoroi-extension/app/components/wallet/assets/AssetsList.js]: Please either access all 3 parameters from `this.props`, or pass them all as parameters.

File: packages/yoroi-extension/app/components/wallet/assets/AssetsList.js [hunk 3]
```diff
@@ -131,10 +142,50 @@ export default class AssetsList extends Component<Props, State> {
...
+    if (request.amount == null) {
+      return <div className={styles.isLoading} />;
+    }
+
+    const defaultEntry = request.amount.getDefaultEntry();
+    const tokenInfo = this.props.getTokenInfo(defaultEntry);
+    const shiftedAmount = defaultEntry.amount
+      .shiftedBy(-tokenInfo.Metadata.numberOfDecimals);
+
+      let balanceDisplay;
```
Comment (yushih) [inline packages/yoroi-extension/app/components/wallet/assets/AssetsList.js]: Please fix indentations.

**Review by yushih — CHANGES_REQUESTED** [review]
LGTM expect for some style problems.

**Review by yushih — APPROVED** [review]

**Review by zuzunker — APPROVED** [review]

**Pull request merged**